src/lwc/geListView/geListView.js

```diff
@@ -315,15 +316,17 @@ export default class geListView extends LightningElement {
                 const columnEntry = this.columnEntriesByName[this.sortedBy];
                 let orderedByFieldApiName;
 
-                if (columnEntry) {
+                if (columnEntry && !this._orderBy) {
                     if (hasNestedProperty(this.columnEntriesByName[this.sortedBy],
                         'typeAttributes', 'label', 'fieldName')) {
                         orderedByFieldApiName = columnEntry.typeAttributes.label.fieldName;
                     } else {
                         orderedByFieldApiName = columnEntry.fieldApiName;
                     }
-                    orderBy = `${orderedByFieldApiName} ${this.sortedDirection}`;
+                } else {
+                    orderedByFieldApiName = this._orderBy;
```

Comment:
Should this be an inner if block?  Like
if (columnEntry) {
  if (!this._orderBy) {
  } else {
  }
}
Would columnEntry ever be false?

Reply:
It can be in an inner block, `columnEntry` will never be false, but `_orderBy` can be `undefined`. I thought it would be cleaner if we combined both if statements. If it affects readability we can change it.